.github/workflows/rspec-shared.yml

```diff
@@ -41,7 +44,7 @@ jobs:
           cpln profile create default --token $CPLN_TOKEN_CI --org $CPLN_ORG
           cpln image docker-login
       - name: Run tests
-        run: bundle exec rspec --format documentation --tag ${{ inputs.test-tag }}
+        run: bundle exec rspec --format documentation ${{ inputs.test-tag && format('--tag {0}', inputs.test-tag) }} ${{ inputs.spec_paths }}
```

Comment:
Where does `format` come from?

Reply:
It's a built-in function which can be used in expressions - https://docs.github.com/en/actions/learn-github-actions/expressions#format